src/hooks/useArbTokenBridge.ts

```
@@ -194,13 +209,31 @@ export const useArbTokenBridge = (
 
   const withdrawEth = useCallback(
     async (etherVal: string) => {
-      if (!arbWallet) throw new Error('withdrawETH no arb wallet')
+      if (!arbWallet || !arbProvider || !walletAddress)
+        throw new Error('withdrawETH no arb wallet')
 
       const weiValue: utils.BigNumber = utils.parseEther(etherVal)
       try {
         const tx = await arbWallet.withdrawEthFromChain(weiValue)
+
         const receipt = await tx.wait()
         updateEthBalances()
+        const { hash } = tx
+        if (!hash) return
```

this should throw an error